File: February 23/17 Feb/Is it Fibonacci gfg.cpp
long long solve(int N, int K, vector<long long> GeekNum) {
        // code here
        vector<long long> series(GeekNum.begin(), GeekNum.end());
        if(N<=K) return series[N-1];
        long long next=0;
        for(int i=0;i<K;i++)
            next+=series[i];
        for(int i=K;i<N;i++)
        {
            series.push_back(next);
            next+=next;
            next-=series[i-K];
        }
        return series.back();
    }

File: February 23/17 Feb/Is it Fibonacci gfg.py
    def solve(self, N, K, GeekNum):
        #your code goes here
        series = GeekNum
        if(N<=K):
            return series[N-1]
        
        sum =0
        for i in range(len(GeekNum)):
            sum = sum + series[i]
        
        for i in range(K,N):
            series.append(sum)
            sum = sum + sum
            sum = sum - series[i-K]
        
        return series[N-1]
